tools/src/h5diff/h5diff_common.c

```diff
@@ -76,7 +76,6 @@ check_options(diff_opt_t *opts)
     }
 }
 
-#if TRILABS_227
 /*-------------------------------------------------------------------------
```

Comment:
Is this macro always defined now?

Reply:
No macro, just a code holder.

Reply:
So that code is active now?

Reply:
yes, most of it has been in for awhile, but I need to test it more widely.

Reply:
OK

Were you defining TRILABS_227 by hand then?  Or was it in a configure option?  Or ???

Reply:
It was a way to keep the code for the release and come back to it.

Reply:
Also, there was a refactor of h5diff functions that was triggered by the work and I just needed to merge for the release without losing working code.

Reply:
Was confusing (obviously! :-) - maybe put it on a branch next time?